test/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(TuriTest)
 enable_testing()
 
 subdirs(
-        annotation
+  # annotation
```

Comment:
Could you explain the changes in this file? I'm not sure I follow.

Reply:
accidentally included for debugging cyclic ninja build. Will be removed.

Reply:
Any reason not to remove it now